app/assets/javascripts/incidents/batch_export.coffee

```diff
@@ -19,22 +19,31 @@ handleExporting = ->
     if alreadyExported > 0
       unless confirm alreadyExportedWarning(alreadyExported)
         return false
-     window.location.href = 'incidents/batch_hastus_export?' + $.param(ids: ids)
-  else
-    $('table.incidents .batch-hastus-export').show()
-    $('.batch-hastus-export #main-button').prop 'disabled', true
-    $(this).text 'Select incidents to export...'
-    $('.batch-hastus-export #select-all').show()
-    setSelectAllText()
+    format = $(this)[0].id
```

Comment:
This is bs to me, but I'm not familiar with the techniques of coffeescript (js, really)